olp-cpp-sdk-dataservice-read/src/generated/api/BlobApi.h

```diff
@@ -39,7 +41,9 @@ namespace read {
  */
 class BlobApi {
  public:
-  using DataResponse = client::ApiResponse<model::Data, client::ApiError>;
+  using DataResponseWithNetworkStats =
+      ExtendedApiResponse<model::Data, client::ApiError,
+                          client::NetworkStatistics>;
```

Comment:
Why did you change it? Just leave DataResponse, it doesn't matter if there is additional information inside.